docs/interop.md

```diff
@@ -0,0 +1,51 @@
+# Interoperability via Scripting
+
+Token SDK supports interoperability, such as HTLC mechanism (see related section), via scripting. 
+It allows spending a token to a script by encoding the script in the `Owner` field of a `Token`, and the different drivers are capable of interpreting the owner as a script.
+
+## HTLC (exchange)
```

Comment:
why do you mean here with exchange? Can you elaborate on this in the paragraph?

Reply:
if I read exchange .... I would assume to read something about two parties that want to exchange tokens ...